packages/office-addin-unit-test/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "office-addin-unit-test",
+  "version": "1.0.0",
```

Comment:
It's not version one until is ready to publish for prime time (and I'm guessing it's not).  should probably do 0.0.1